18_links_lab/linkslab.txt

```diff
@@ -2,19 +2,32 @@ Preparation: Perform the following  (no need to write the commands you used):
 
   - create a new directory named "links"
   - cd to it
-  - inside the directory create 2 text files named "file1" and "file2". Write the names of some things you like in file "file1", and some things you don't like in file "file2"
+  - inside the directory create 2 text files named "file1" and "file2". 
+Write the names of some things you like in file "file1", and some things you don't like in file "file2"
 
+
 1. Perform the following and write the commands you used, from within links directory:
 
   - Create a symbolic link named file3 that points to "file1"
-  Answer: 
+  
+Answer: 
 
-  - Create a hard link named file4 that points to "file2"
-  Answer: 
+ ln -s file1 file3
+- Create a hard link named file4 that points to "file2"
+  
+Answer: 
 
-  - Create a new directory named "tmp" in your home folder:
-  Answer:
+  ln file2 file4
+- Create a new directory named "tmp" in your home folder:
+  
+Answer:
 
-2. Move "file3" to the new "~/tmp" directory you just created. Can you still use it to see the original "file1" ? Why not? How would you create a symbolic link that would survive the transfer to ~/tmp ?
+   mkdir /home/avia/tmp
+
+2. Move "file3" to the new "~/tmp" directory you just created. Answer: mv home/avia/lab4/links/file3 /home/avia/tmp
+Can you still use it to see the original "file1" ? No. Why not? Because when moving synbolic link to different directory - the link will not work for the original file anymore.
+How would you create a symbolic link that would survive the transfer to ~/tmp ?
 
-3. How can you tell that "file2" and "file4" refer to the same file?
+ln -s /home/avia/lab4/links/file1 /home/avia/tmp/file4
+
+3. How can you tell that "file2" and "file4" refer to the same file? ls -l gives us output for the files that exist in the folder, which we can see that file 2 and file 4 have the same date+permissions+ 2 = numbers of names to the same file
```

Comment:
The magic switch is `-i` - it shows a special unique index of a file that is only equal if the files are hard links to one another. Use `ls -li file1 file2 file3 file4` and watch the inode number, you'll see files 2 and 4 have the same